checkout/javascript/cart-checkout-validation/src/index.liquid

```diff
@@ -0,0 +1,56 @@
+{%- if flavor contains "vanilla-js" -%}
+// @ts-check
+
+/**
+ * @typedef {import("../generated/api").InputQuery} InputQuery
+ * @typedef {import("../generated/api").FunctionResult} FunctionResult
+ */
+
+/**
+ * @type {FunctionResult}
+ */
+export default /**
+ * @param {InputQuery} input
+ * @returns {FunctionResult}
+ */
+function cartCheckoutValidation(input) {
+  const errors = []
```

Comment:
Should be able to simplify this with something like `return car.lines.filter(<quantity greater than 1>).map(<error>)`